### PR TITLE
Suggest french translation refactor for Base readme 

### PR DESCRIPTION
**Pull request by ManuSquall**
Refactor and update of the base readme french translation

### CODE REVIEW

**Review by jlooper — CHANGES_REQUESTED** [review]
just a few comments

File: translations/README.fr.md [hunk 1]
```diff
@@ -8,108 +8,97 @@
...
 
+[![Open in Visual Studio Code](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/microsoft/Web-Dev-For-Beginners)
 
 # Développement Web pour débutants - Programme
 
-Azure Cloud Advocates de Microsoft sont heureux de vous offrir un cours de 24 leçons portant sur les bases du HTML, CSS et JavaScript.
-Chaque leçon comporte des instructions, des projets et des quizs.
-Grâce à la pédagogie basée sur la pratique, vous pouvez apprendre tout en créant des interfaces.
-Ce qui représente une méthode qui consiste à acquérir des compétances et s'en rappelant !.
+Azure Cloud Advocates de Microsoft a le plaisir de proposer un programme de 12 semaines et 24 leçons sur les bases de JavaScript, CSS et HTML. Chaque leçon comprend des quiz avant et après la leçon, des instructions écrites pour terminer la leçon, une solution, un devoir et plus encore. Notre pédagogie basée sur des projets vous permet d’apprendre tout en construisant, un moyen éprouvé pour que de nouvelles compétences puissent "coller".
```
[the remark this answers is not in the document]
Comment (ManuSquall) [reply translations/README.fr.md:15]: I have replaced "Azure Cloud Advocates de Microsoft a le plaisir de proposer" with "Azure Cloud Advocates de Microsoft ont le plaisir de vous proposer"; hope it fits better ?

File: translations/README.fr.md [hunk 2]
```diff
@@ -8,108 +8,97 @@
...
 
-**Un très grand merci à Jen Looper, Chris Noring, Christopher Harrison, Jasmine Greenaway, Yohan Lasorsa, Floor Drees, et Tomomi Imura!**
+**Un grand merci à nos auteurs Jen Looper, Chris Noring, Christopher Harrison, Jasmine Greenaway, Yohan Lasorsa, Floor Drees et l’artiste de sketchnote Tomomi Imura!**
 
 # Pour commencer
 
-> **Instituteurs**, on vous a préparé des [suggestions](for-teachers.md). On attend votre retour sur ce cours au [forum](https://github.com/microsoft/Web-Dev-For-Beginners/discussions/categories/teacher-corner)!
+> **Instituteurs**, nous avons [inclus quelques suggestions] (pour teachers.md) sur la façon d’utiliser ce programme. Nous aimerions avoir vos commentaires [dans notre forum de discussion] (https://github.com/microsoft/Web-Dev-For-Beginners/discussions/categories/teacher-corner)!
```
[the remark this answers is not in the document]
Comment (ManuSquall) [reply translations/README.fr.md:21]: Yes, totally agree. I forgot to link the relative for-teachers.fr.md file. I also delete the space after "dans notre forum de discussion]" to make the link work. Thanks you.

**Comment by ManuSquall** [comment]
> just a few comments

I have made some changes according to that, hope it will be good, thanks for the review 🥳!

**Pull request merged**